.github/workflows/daily.yaml

```diff
@@ -1,36 +1,35 @@
-name: Daily Test
+name: Daily Integration Test
 
 on:
-  workflow_dispatch: # run on request (no need for PR)
+  workflow_dispatch: # Allows manual trigger
   schedule:
-    # every UTC 7PM from Mon to Fri
-    - cron: "0 19 * * 1-5"
+    # every UTC/GMT 8PM from Mon to Fri
+    - cron: "0 20 * * 1-5"
 
 # Declare default permissions as read only.
 permissions: read-all
 
 jobs:
-  Daily-e2e-tests:
-    runs-on: [otx-gpu-a10g-1]
-    container:
-      image: 219678651685.dkr.ecr.eu-west-1.amazonaws.com/ote-ci:11.7.1.2-devel-ubuntu20.04
-      options: "--runtime=nvidia --env-file=/home/runner/.nvidia.env --ipc=host"
+  Daily-Integration-Tests:
+    runs-on: [otx-ci-3090-03-all]
     strategy:
       fail-fast: false
       matrix:
         include:
-          - task: "classification"
-          - task: "detection"
-          - task: "rotated_detection"
+          - task: "multi_cls_classification"
+          - task: "multi_label_classification"
+          - task: "hlabel_classification"
+          - task: "visual_prompting"
+          - task: "anomaly_classification"
```

Comment:
I would rather remove all anomaly_* and leave only anomaly. Since other tasks are deprecated in new Geti version. Also, I am not sure about visual prompting, but let's keep it for now.

Reply:
yeah, we can remove those tests while removing the anomaly task. But I don't think we should do it in this PR. This PR is mainly changing our daily.